src/python/pants/backend/jvm/tasks/coursier_resolve.py

```diff
@@ -57,7 +57,7 @@ def register_options(cls, register):
     super(CoursierMixin, cls).register_options(register)
     register('--allow-global-excludes', type=bool, advanced=False, fingerprint=True, default=True,
              help='Whether global excludes are allowed.')
-    register('--report', type=bool, advanced=False, fingerprint=True, default=False,
+    register('--report', type=bool, advanced=False, fingerprint=False, default=False,
```

Comment:
False is the default, so unless you're leaving this here as a "comment" indicating that this is intentionally not fingerprinted, would remove it. Although a comment wouldn't hurt either.

Reply:
`fingerprint=False` and added the behavior to help msg